### PR TITLE
Portfolio project with Microverse

### PR DESCRIPTION
**Pull request by engrabdullahriaz**
Portfolio: setup and mobile version skeleton
Added all required files to the repo and made commit to repo.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  padding: 0;
+}
+
+body {
+  margin: 10px 8px;
+  background: transparent;
+  font-family: sans-serif;
+  background-color: #f7f7f9;
+  background-image: url(images/Headerbg.png);
+}
```
Comment (smunozmo) [inline style.css]: - [ ] You are almost there! However, your whole `body` is using `sans-serif` for texts, let's use Poppins as shown in the Figma template.

- [ ] Nice job using CSS to set your section's background, but you are using the desktop version background which is different from the mobile version background, you can find it on the left sidebar on the Figma app.

Desktop bg:
Captura de pantalla 2022-04-23 a la(s) 12 08 17 a  m

Mobile bg:
Captura de pantalla 2022-04-23 a la(s) 12 08 32 a  m

File: index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      <div class="icons-header">
+        <h2>LET'S CONNECT</h2>
+      </div>
+      <ul class="icons">
+        <li><a href="#" class="icon-link"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-angellist" aria-hidden="true"></i></a></li>
+      </ul>
```
Comment (smunozmo) [inline index.html]: - [ ] Icons are looking great! Let's use the color from the template:

Template:
Captura de pantalla 2022-04-22 a la(s) 11 57 55 p  m


Your design:
Captura de pantalla 2022-04-22 a la(s) 11 58 07 p  m

File: index.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  <!-- start of nav section-->
+  <nav class="nav-wrapper">
+    <!--nav left side-->
+    <div class="nav-left">
+      <a href="#">My Logo</a>
+    </div>
+    <!-- nav right side-->
+    <div class="nav-right">
+        <button class="hamburger"><i class="fa fa-bars"></i></button>
+    </div>
```
Comment (smunozmo) [inline index.html]: - [ ] Great work aligning your hamburger menu button, well done! Let's make it even better by using the template's color and removing the surrounding border which is not part of the design, try with `border` or `outline` properties to remove it:

Template:
Captura de pantalla 2022-04-23 a la(s) 12 10 16 a  m


Your design:
Captura de pantalla 2022-04-23 a la(s) 12 10 24 a  m

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @engrabdullahriaz, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (engrabdullahriaz) [reply style.css]: Okay now I am using Poppins.

**Pull request merged**